packages/vats/index.js

```diff
@@ -0,0 +1,9 @@
+// @ts-check
+// Ambient types
+import './src/core/types.js';
+
+export * from './src/core/boot.js';
+// eslint-disable-next-line import/export -- no named exports
+export * from './src/types.js';
+export * from './src/nameHub.js';
+export * from './src/bridge.js';
```

Comment:
I still believe this will cause formerly internal runtime helpers to now be visible on the root of this package, not only types.

See https://github.com/Agoric/agoric-sdk/pull/6500#discussion_r1008709131

Reply:
Thanks for the thorough review!

> I suspect the ./bridge.js import is for the BridgeManager type? In that case I believe the better approach would be to have a JSDoc @typedef in ../types.js and leverage the existing re-export from there. Same the nameHub.js's NameHub and NameAdmin.

`makeNameHubAdmin` is imported in three other modules, so arguably this top level is an improvement, but I'll err towards conservative and leave runtime exports refactoring for another day.